.gitlab-ci.yml

```diff
@@ -2,7 +2,7 @@ variables:
   # These are gitlab variables so that it's easier to do a manual deploy
   # If these are set witih value and description, then it gives you UI elements
   DOWNSTREAM_BRANCH:
-    value: "main"
+    value: "levi/bye-ddprof-ffi"
```

Comment:
Might need to remove this before merging I guess.

Reply:
Just merged the branch; will push the change now. Thanks!